crates/subspace-networking/src/create.rs

```diff
@@ -80,7 +80,7 @@ const SWARM_MAX_ESTABLISHED_CONNECTIONS_PER_PEER: Option<u32> = Some(2);
 const ENABLE_GOSSIP_PROTOCOL: bool = false;
 //TODO: Investigate port reuse for QUIC
 /// Specifies whether we use the same outgoing port for TCP connections.
-const TCP_PORT_REUSE: bool = true;
+const TCP_PORT_REUSE: bool = false;
```

Comment:
Why changing to `false`?

Reply:
This feature seems to be mutually exclusive with the address translation. There is no observed effect after enabling it. I would discuss that flag with libp2p team before toggling it back.

Reply:
Address translation helps node discover likely address through which it is reachable, while port reuse, depending on how NAT is configured, can help that port actually match the expectation. So I'm not sure why you're saying they are exclusive, they seem to be complementary instead.

Reply:
Ok. I toggled it back and would ask about side-effects.